buildlib/azure-pipelines-perf.yml

```diff
@@ -106,11 +109,43 @@ stages:
                 --after "$SHA_After" \
                 --module hpcx-gcc \
                 --filter \
-                --extra-ompi-opts " -H $(perf_nodes)" | tee $(WorkDir)/results.txt
-            displayName: Perf test
+                --omb-cuda \
+                --extra-ompi-opts " -H $(perf_nodes)" | tee $(WorkDir)/results-multinode.txt
+            displayName: Perf test - multi node
             workingDirectory: $(WorkDir)
 
-          - script: $(WorkDir)/ucx/buildlib/tools/perf_results.py $(WorkDir)/results.txt $(threshold)
+          - bash: |
+              set -eE
+              SHA_After=$(echo $(Build.SourceVersionMessage) | awk '{print $2}')
+              SHA_Before=$(echo $(Build.SourceVersionMessage) | awk '{print $4}')
+
+              python3 -m venv VENV
+              source VENV/bin/activate
+              cd $(WorkDir)/PerfX
+              module load /hpc/local/etc/modulefiles/hpcx-gcc
+
+              set -xeE -o pipefail
+              # Pick random node
+              IFS=',' read -ra nodes <<< "$(perf_nodes)"
+              node=${nodes[RANDOM % 2]}
+
+              python3 -u perfx.py \
+                --skip-update \
+                --skip-build \
+                --use-cache \
+                --source-branch $(Build.SourceBranch) \
+                --config ucx-rdmz.yml \
```

Comment:
I think that the config should be stored in the place where perfx should be used. Otherwise, we should update perfx repo, if we want to change configuration of UCX tests. Moreover, you should check perfx repo to find out testing parameter, e.g. the list of benchmarks.

I know that this comment is not related to this PR, because it was added earlier. However, it can be discussed and addressed in the future.

Reply:
Adding the PerfX architect: @amaslenn

Reply:
I agree with @rakhmets.

The only thing I worry about in this case is compatibility: I don't understand how Perfx is delivered to this pipeline and it might happen that it doesn't support new config option and maybe it dropped support for another.

Explicit git checkout with a tag should solve it though.

Reply:
There is an explicit checkout:
```
resources:
  repositories:
  - repository: PerfX
    type: github
    name: Mellanox-lab/PerfX
    endpoint: Mellanox-lab
    ref: ucx_extend
```